modules/tools/diagnostics/message.py

```diff
@@ -93,9 +93,9 @@ def callback(self, data):
             time = 0
             sequence_num = 0
 
-        if self.msg_received == True:
+        if self.msg_received:
             seq_diff = sequence_num - self.sequence_num
-            if seq_diff is not 0:
+            if seq_diff == 0:
```

Comment:
It is not correct.

Reply:
Um..., thanks for your review!  That's my mistake, will fix it ASAP.